crocket/src/main/java/com/crocket/DrawableEntity.java

```diff
@@ -39,4 +40,10 @@ public int getRotation() {
     public EntityType getType() {
         return type;
     }
+
+    public String getTypeName() {
+        return type.name();
```

Comment:
Is this one necessary? Why call for EntityType methods in DrawableEntity class?

Reply:
Yes it might seem weird but it is a result of the texture manger class.  Essentially, we utilize a JSON file (JSON does not have enums) to map strings to corresponding textures. The strings share the same names as the enums. To enhance readability and adhere to the Law of Demeter, we use ```getTypeName()``` instead of ```entity.getType().name()```.